user_front/pages/mypage/publicRelations/index.vue

```diff
@@ -93,7 +93,7 @@ const fileUpload = (e: Event) => {
 const fileCheck = () => {
   if (selectedFile.value) {
     // jpeg,pngの指定、それ以外の場合はエラー
-    if (!["image/jpeg", "image/png"].includes(selectedFile.value.type)) {
+    if (!["image/jpeg", "image/png", "image/png"].includes(selectedFile.value.type)) {
```

Comment:
```suggestion
    if (!["image/jpeg", "image/png"].includes(selectedFile.value.type)) {
```
同じの追加されてるから元に戻していいかも